receiver/flinkmetricsreceiver/metadata.yaml

```diff
@@ -0,0 +1,319 @@
+name: flinkmetricsreceiver
+
+resource_attributes:
+  # These resource attributes are Flinks system scope variables, which contains context information about metrics. These are required to uniquely identify incoming metrics as the same job can run multiple times concurrently. See https://nightlies.apache.org/flink/flink-docs-release-1.14/docs/ops/metrics/#system-scope for more information.
+  host.name:
+    description: The host name.
+    type: string
+  flink.taskmanager.id:
+    description: The taskmanager ID.
+    type: string
+  flink.job.name:
+    description: The job name.
+    type: string
+  flink.task.name:
+    description: The task name.
+    type: string
+  flink.subtask.index:
+    description: The subtask index.
+    type: string
+  flink.resource.type:
+    description: The flink scope type in which a metric belongs to.
+    type: string
+
+attributes:
+  operator_name:
+    description: The operator name.
+    type: string
+  garbage_collector_name:
+    description: The names for the parallel scavenge and garbage first garbage collectors.
+    type: string
+    enum: [ PS_MarkSweep, PS_Scavenge, G1_Young_Generation, G1_Old_Generation ]
+  checkpoint:
+    description: The number of checkpoints in progress, completed or that failed. 
+    type: string
+    enum: [ in_progress, completed, failed ]
+  record:
+    description: The number of records received in, sent out or dropped due to arriving late.
+    type: string
+    enum: [ in, out, late_records_dropped ]
+
+metrics:
+  flink.jvm.cpu.load:
+    enabled: true
+    description: The recent CPU usage of the JVM.
+    unit: 1
+    gauge:
+      value_type: double
+      input_type: string
+    attributes: []
```

Comment:
Assuming this is linux cpu load, this is typically reported as values representing the last 1, 5, and 15 minutes. Can we specify which one this is, or maybe include all three with an attribute?

Reply:
I believe you are referring to [system resources metrics](https://nightlies.apache.org/flink/flink-docs-release-1.14/docs/ops/metrics/#system-resources) that include a load1min, load5min, and load15min. This is disabled by default, so these metrics where not being collected as of this implementation. The JVM CPU Load metric changes every 10 seconds. I think this description is fine unless you want `The CPU usage of the JVM since the last 10 seconds`.

Reply:
The problem here is that it's not clear what "load" means, especially without units.

There's a somewhat standard definition of load in linux, which is unitless. If this isn't that, then it needs a clearer description.

Reply:
I found [this](https://www.adfpm.com/adf-performance-monitor-mayor-new-version-9-0-part-1/) article that has a nice visual showing the CPU load % of the `jvm` process and the whole operating `system` (second image).  I think load is the same load as you are used to, but in the context of the an application (jobmanager/taskmanager) cpu % confined by the jvm process. I'll add in a % in the description. I did not see other metadata files that had a percentage as unit